src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -104,12 +104,18 @@ object GermlineAssemblyCaller {
       val referenceKmerSink = currentReference.takeRight(kmerSize)
 
       // If the current window size doesn't cover the kmer size we won't be able to find the reference start/end
-      if (currentReference.size < kmerSize)
+      if (currentReference.size < kmerSize || referenceKmerSource == referenceKmerSink)
```

Comment:
OK, punting wfm. I see that there is a more general inability to handle regions with repeating k-mers atm. I wonder if that happens frequently? Adding a TODO and/or a warning about this case might make sense.

Basically any time we skip a node in the DBG DFS due to it already being in `visited`, we are discarding a possible valid path with a repeat segment in it, right?